lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +23,18 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if type(text) == str:
+        text = text.lower().split()
+        split2 = []
+        for word in text:
+            for token in word:
+                if not token.isalpha():
+                    continue
+                else:
```

Comment:
else is not required, enough write split2 += token

Reply:
fixed